sdcm/tester.py

```diff
@@ -874,6 +900,7 @@ def setUp(self):  # pylint: disable=too-many-branches,too-many-statements
             if db_cluster:
                 db_cluster.validate_seeds_on_all_nodes()
                 validate_raft_on_nodes(nodes=db_cluster.nodes)
+                db_cluster.start_kms_key_rotation_thread()
```

Comment:
can get add SCT configuration for setting `rotation_interval_m` ? i'm not sure a default 60m would be suitable for all cases ?

Reply:
Done